src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,125 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
Replace `const centuries = searchParams.getAll('centuries') || [];` with `const centuries = searchParams.getAll('centuries');` because `getAll` method always returns an array, even if it's empty. 🧹